keras_cv/models/object_detection/retina_net/__internal__/layers/prediction_head.py

```diff
@@ -64,9 +64,22 @@ def call(self, x, training=False):
 
     def get_config(self):
         config = {
-            "bias_initializer": self.bias_initializer,
+            "bias_initializer": keras.initializers.serialize(
```

Comment:
Does this mean that we won't be able to serialize the string "zeros", and the actual keras initializer will have to be passed?

Reply:
Yes. I see `PredictionHead` as a internal only class, so we may enforce it. If we still want the "zeros", we  can do a special case here for the string types.